src/index.html

```diff
@@ -17,11 +17,344 @@
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__info">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__review">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#buy"
+          class="product-card__buy"
+        >
+          Buy
+        </a>
+      </article>
+      <article class="product-card">
+        <img
+          class="product-card__image"
+          src="./images/imac.jpeg"
+          alt="Product logo"
+        />
+        <h3 class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__info">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__review">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#buy"
+          class="product-card__buy"
+        >
+          Buy
+        </a>
+      </article>
+      <article class="product-card">
+        <img
+          class="product-card__image"
+          src="./images/imac.jpeg"
+          alt="Product logo"
+        />
+        <h3 class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__info">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__review">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#buy"
+          class="product-card__buy"
+        >
+          Buy
+        </a>
+      </article>
```

Comment:
All product cards have identical content, including the product name, code, reviews, and price. If these are meant to represent different products, ensure each card has unique content. If they are placeholders, consider adding a comment to clarify this.